values.yaml

```diff
@@ -106,6 +106,9 @@ server:
               component: server
           topologyKey: kubernetes.io/hostname
 
+  # Extra annotations to attach to the server pods
```

Comment:
This comment should include some information about the expected format of the input for the value.